checkbox-core-snap/README.md

```diff
@@ -0,0 +1,75 @@
+# Building checkbox-core-snap
+This snap is the core reusable part of checkbox, it includes all the utilities and some core providers.
+
+## Environment Creation
+This small guide mainly covers offline builds with `--destructive-mode` enabled. These builds must be carried out in single-use containers as the name implies. This guide will explain how to build the `series22` snap, in order to build the others the process is the same, the guide will point out what to change.
```

Comment:
snapcraft without destructive mode uses multipass, that is not super handy/useful for someone trying to build the snaps, especially if they are developing/debugging builds or if they are using a container. Destructive mode allows us to use the host system destructively, saving time on every build. This is explained well in the readme, that instructs the user to always use a container when doing so. I saw there is an [lxd backend ](https://snapcraft.io/blog/faster-snap-development-additional-tips-and-tricks) but I do not know if it can be used for our means and if it can be used to emulate what we need for speedy debugging. The point is that with destructive mode we install the build dependencies in the container, and that saves a lot of time on re-runs.

Reply:
Using destructive mode shouldn't be the default. The time difference is not big tbh, as multipass reuses the vm.

Reply:
Don't you think that using destructive mode makes it easier/faster to get going and debug?